android/recipes/src/main/java/kmp/android/recipes/navigation/RecipesDestination.kt

```diff
@@ -0,0 +1,34 @@
+package kmp.android.recipes.navigation
+
+import androidx.annotation.StringRes
+import kmp.android.shared.Feature
+
+sealed class RecipesDestination(
+    val route: String,
+    @StringRes val name: Int,
+    @StringRes val description: Int,
+) {
+    companion object {
+        val root: String = Feature.Recipes.route
+    }
+
+    object List : RecipesDestination("$root/list", 0, 0)
```

Comment:
Do not use 0 as a default value, if someone tried to use it not knowing it's set to 0, the app would not find the resource and crash, instead you can make `name` and `description` nullable and set it to `null` here.